CODEOWNERS

```diff
@@ -1 +1 @@
-* @brikis98 @yorinasub17
+@yorinasub17 @zackproser @rhoboat
```

Comment:
Same question as `terragrunt-infrastructure-modules-example`, about whether @denis256 should be added here.